packages/showcases/css/stories/components/toggle/examples/default.html

```diff
@@ -0,0 +1,7 @@
+<div class="block">
+  <label class="vtmn-toggle_switch">
+    <input type="checkbox" class="vtmn-toggle" />
+    <span class="vtmn-toggle_slider"></span>
+  </label>
+</div>
```

Comment:
Maybe we should avoid nesting the input inside a dedicated label.
This will be problematic when implementing this next to  proper label like :
- [ ] Toggle me

"Toggle me" will have to be a label targeting '.vtmn-toggle'

Though it seems valid to have several label for the same input, it messes assistive technologies. 

Maybe we should dig something [this way](https://codepen.io/__sch__/pen/rNjYWxM?editors=1100) (keeping the input below the slider and keeping it clickable with a combination of opacity and / or pointer-events properties)

Reply:
Thanks for telling about accessibility, I didn't pay attention about this one.